datahub-web-react/src/app/ingest/source/builder/constants.ts

```diff
@@ -0,0 +1,145 @@
+export const TABLEAU = 'tableau';
+export const MYSQL_URN = 'urn:li:dataPlatform:mysql';
+export const MYSQL = 'mysql';
+export const POSTGRES_URN = 'urn:li:dataPlatform:postgres';
+export const POSTGRES = 'postgres';
+export const MONGO_DB_URN = 'urn:li:dataPlatform:mongodb';
+export const MONGO_DB = 'mongodb';
+export const AZURE_URN = 'urn:li:dataPlatform:azure-ad';
+export const AZURE = 'azure-ad';
+export const OKTA_URN = 'urn:li:dataPlatform:okta';
+export const OKTA = 'okta';
+export const GLUE_URN = 'urn:li:dataPlatform:glue';
+export const GLUE = 'glue';
+export const ORACLE_URN = 'urn:li:dataPlatform:oracle';
+export const ORACLE = 'oracle';
+export const HIVE_URN = 'urn:li:dataPlatform:hive';
+export const HIVE = 'hive';
+export const SUPERSET_URN = 'urn:li:dataPlatform:superset';
+export const SUPERSET = 'superset';
+export const ATHENA_URN = 'urn:li:dataPlatform:athena';
+export const ATHENA = 'athena';
+export const MSSQL_URN = 'urn:li:dataPlatform:mssql';
+export const MSSQL = 'mssql';
+export const CLICKHOUSE_URN = 'urn:li:dataPlatform:clickhouse';
+export const CLICKHOUSE = 'clickhouse';
+export const TRINO_URN = 'urn:li:dataPlatform:trino';
+export const TRINO = 'trino';
+export const DBT_URN = 'urn:li:dataPlatform:dbt';
+export const DBT = 'dbt';
+export const DRUID_URN = 'urn:li:dataPlatform:druid';
+export const DRUID = 'druid';
+export const ELASTICSEARCH_URN = 'urn:li:dataPlatform:elasticsearch';
+export const ELASTICSEARCH = 'elasticsearch';
+export const FEAST_URN = 'urn:li:dataPlatform:feast';
+export const FEAST = 'feast';
+export const MARIA_DB_URN = 'urn:li:dataPlatform:mariadb';
+export const MARIA_DB = 'mariadb';
+export const METABASE_URN = 'urn:li:dataPlatform:metabase';
+export const METABASE = 'metabase';
+export const CUSTOM_URN = 'urn:li:dataPlatform:custom';
+export const CUSTOM = 'custom';
+export const POWER_BI_URN = 'urn:li:dataPlatform:powerbi';
+export const POWER_BI = 'powerbi';
+export const MODE_URN = 'urn:li:dataPlatform:mode';
+export const MODE = 'mode';
+
+export const SOURCE_URN_TO_LOGO = {
+    [SNOWFLAKE_URN]: snowflakeLogo,
+    [BIGQUERY_URN]: bigqueryLogo,
+    [REDSHIFT_URN]: redshiftLogo,
+    [KAFKA_URN]: kafkaLogo,
+    [LOOKER_URN]: lookerLogo,
+    [TABLEAU_URN]: tableauLogo,
+    [MYSQL_URN]: mysqlLogo,
+    [POSTGRES_URN]: postgresLogo,
+    [MONGO_DB_URN]: mongodbLogo,
+    [AZURE_URN]: azureLogo,
+    [OKTA_URN]: oktaLogo,
+    [GLUE_URN]: glueLogo,
+    [ORACLE_URN]: oracleLogo,
+    [HIVE_URN]: hiveLogo,
+    [SUPERSET_URN]: supersetLogo,
+    [ATHENA_URN]: athenaLogo,
+    [MSSQL_URN]: mssqlLogo,
+    [CLICKHOUSE_URN]: clickhouseLogo,
+    [TRINO_URN]: trinoLogo,
+    [DBT_URN]: dbtLogo,
+    [DRUID_URN]: druidLogo,
+    [ELASTICSEARCH_URN]: elasticsearchLogo,
+    [FEAST_URN]: feastLogo,
+    [MARIA_DB_URN]: mariadbLogo,
+    [METABASE_URN]: metabaseLogo,
+    [POWER_BI_URN]: powerbiLogo,
+    [MODE_URN]: modeLogo,
+};
+
+export const SOURCE_TO_SOURCE_URN = {
```

Comment:
this feels wrong to me 🤔 if we are already encoding this mapping in mysql why do we need to recreate it in react?